package.nls.json

```diff
@@ -78,6 +79,10 @@
     "AWS.configuration.description.awssam.debug.dockerNetwork": "Specifies the name or id of an existing Docker network that Lambda Docker containers should connect to.",
     "AWS.configuration.description.awssam.debug.localArguments": "Additional arguments to pass to the `sam local` command.",
     "AWS.configuration.description.awssam.debug.skipNewImageCheck": "Specifies whether the command should skip pulling down the latest Docker image for Lambda runtime (default: false).",
+    "AWS.configuration.description.awssam.debug.snippets.lambdaCode.label": "AWS SAM: Direct Lambda Invoke",
```

Comment:
This might be confusing considering we use the `direct-invoke` terminology for both `code` and `template`.